package.json

```diff
@@ -37,6 +37,7 @@
     "react-native": "~0.55.2",
     "react-native-audio": "^4.1.3",
     "react-native-config": "^0.11.5",
+    "react-native-sound-recorder": "^1.3.2",
```

Comment:
Are we planning to use this package just for Android? `react-native-audio` is doing the sound recording on iOS, or are we planning to change that?